### DISCUSSION TITLE
multi-time series forecasting in production

### DISCUSSION THREAD

**Discussion by angelicarebolini** [JoaquinAmatRodrigo/skforecast#406] [General]
¿Cómo se puede utilizar en producción un modelo autoregresivo entrenado con múltiples series de tiempo? 
Por ejemplo, si se baja un producto de la serie porque ya no está disponible o quiero pronosticar otros productos que no estaban en la serie entrenada, ¿se puede hacer? He intentado esto último y obtuve el siguiente error "_ValueError: `last_window` columns must be the same as `series` column names"_

**Comment by JoaquinAmatRodrigo** [comment]
Hola @angelicarebolini,
Muchas gracias por utilizar skforecast :)

Los modelos no permiten hacer predicciones sobre series que no han participado durante el entrenamiento. Esto se ha programado así dado que el modelo no tiene ninguna evidencia para asumir que el comportamiento de la nueva serie es igual que el de las demás. Se trata de un problema de "transfer learning".

Saludos!

**Reply by angelicarebolini** [reply]
Gracias por la aclaración. Saludos1
